content/news/buzz/november-2025-status-update/contents.lr

```diff
@@ -0,0 +1,46 @@
+- We completed a large refactor of BeeWare's documentation. The BeeWare website [now has a clear "Documentation" heading](https://github.com/beeware/beeware.github.io/pull/716), and all of BeeWare's main projects serve their documentation from `beeware.org` subdomains.
+- We expanded the BeeWare tutorial to include a section on [runtime icon handling](https://github.com/beeware/beeware-tutorial/pull/35)
+- We completed an update to Chaquopy, the tooling underpinning Briefcase's Android support, adding support for Python 3.14, as well as a number of other updates to adapt to changes in the Android ecosystem.
+- We added [integration with PDB and VSCode debuggers for Briefcase apps](https://github.com/beeware/briefcase/pull/2173). This allows users to use their IDE to debug Windows, Linux, macOS and iOS apps. Support for Android should be added soon. Support for PyCharm and other debuggers should also be possible given the debugging framework that has been put in place.
+- We added the ability to [customize the install directory](https://github.com/beeware/briefcase/pull/2567), and [configure pre-uninstall options](https://github.com/beeware/briefcase/pull/2582) in Windows MSI installers.
+- We modified Briefcase to make [local source references in a development environment editable installs](https://github.com/beeware/briefcase/pull/2498)
+- We added support for [a cross-platform Bluetooth permission](https://github.com/beeware/briefcase/pull/2522) in Briefcase apps.
+- We [improved Python version detection when building apps on Linux](https://github.com/beeware/briefcase/pull/2546)
+- We added a new [Qt backend](https://github.com/beeware/toga/pull/3769) to Toga! This will allow Linux users with a KDE desktop to have fully native apps, instead of relying on GTK theme emulation. The Qt backend is still in development, but it already has support for a lot of simple widgets, including Button, Label, [Switch](https://github.com/beeware/toga/pull/3917) and [ImageView](https://github.com/beeware/toga/pull/3918).
+- We added [Button](https://github.com/beeware/toga/pull/3881), [TextInput](https://github.com/beeware/toga/pull/3239), [Canvas](https://github.com/beeware/toga/pull/3904) [DateInput testing](https://github.com/beeware/toga/pull/3915), and [Activityindicator](https://github.com/beeware/toga/pull/3896) widgets to Toga's GTK4 implementation, as well as adding support for fonts and icons.
+- We added a [handler for window resize events on Toga Windows](https://github.com/beeware/toga/pull/2364)
+- We added support in Toga's DecimalInput for [locales that use a comma as a decimal separator](https://github.com/beeware/toga/pull/3870).
+- We improved Toga's WebView, ensuring that it [can access `SharedArrayBuffer` APIs on GTK](https://github.com/beeware/toga/pull/3927). We also modified the Positron static plugin to set headers that are necessary for 
```

Comment:
You appear to have moved on in the middle of a sentence here.

Reply:
🤦